unetsocket/c/samples/npulses.c

```diff
@@ -0,0 +1,79 @@
+///////////////////////////////////////////////////////////////////////////////
+//
+// Script to transmit a signal.
+//
+// In terminal window (an example):
+//
+// $ make samples
+// $ ./npulses <ip_address> [port]
+//
+////////////////////////////////////////////////////////////////////////////////
+
+#include <stdio.h>
+#include <stdlib.h>
+#include "../unet.h"
+#include "../unet_ext.h"
+#include <math.h>
+
+#ifndef _WIN32
+#include <unistd.h>
+#include <netdb.h>
+#include <sys/time.h>
+#endif
+
+#ifndef M_PI
+  #define M_PI 3.14159265358979323846
+#endif
+
+#define FREQ    5000 // change this as per the need
```

Comment:
Should we make `FREQ` and `SIGLEN` cli arguments? Instead of getting the user to change them in code?

Reply:
Similarly in other samples.